src/_http.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-var APIVersion = '4'
-
+var pjson = require('../package.json')
+var parse = require('url-parse')
```

Comment:
I can be mistaken, but I think you have removed this one in the previous PR

Reply:
done